task_manager/src/lib.rs

```diff
@@ -134,6 +134,7 @@ fn start_futures(
         .collect()
 }
 
+#[allow(clippy::manual_try_fold)]
```

Comment:
why the clippy silence here instead of just switching to a `try_fold`?

Reply:
Oh because I couldn't actually get it to work locally. Clippy kept complaining whichever way I spun it. Feel free to take a look and remove the silencer

Reply:
so i tried this and if i'm understanding the errors correctly, i don't think the compiler is able to determine that the iterator of futures implements TryStream so it won't recognize the `try_fold` but maybe @maplant can finesse the incantation needed

Reply:
i took @maplant 's advice and converted this to a `futures::stream::iter(stream).then().collect().await.into_iter().collect()` since he points out it seems to logically perform the operation we want (await all futures to completion instead of exiting early on the first Err result) and avoids the need to silence any difficult clippy lints